setup.py

```diff
@@ -74,6 +74,7 @@ def local_file(name):
     zip_safe=False,
     extras_require=extras,
     install_requires=install_requires,
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*',
```

Comment:
No ruling against, but neither is there any move to fix broken tests.

I think documenting support for 2.7 & 3.4+ and allowing installs on 3.3 is a good compromise.